applications/StructuralMechanicsApplication/custom_strategies/custom_builder_and_solver/explicit_builder_and_solver.hpp

```diff
@@ -0,0 +1,461 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Klaus B Sautter (based on the work of JMCarbonell)
+//					 
+
+#if !defined(KRATOS_EXPLICIT_BUILDER_AND_SOLVER )
+#define  KRATOS_EXPLICIT_BUILDER_AND_SOLVER
+
+
+/* System includes */
+#include <set>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+/* External includes */
+
+
+/* Project includes */
+#include "includes/define.h"
+#include "solving_strategies/builder_and_solvers/builder_and_solver.h"
+#include "includes/model_part.h"
+#include "includes/element.h"
```

Comment:
do you explicitly have to include this?

Reply:
In fact the model_part.h includes the element.h